ext/kubernetes/deployment/init_test.go

```diff
@@ -0,0 +1,47 @@
+
+		return byteArrayExample, nil
+	}
+
+	outputCircle, _ := triggerRequest("curl -u SOMETHING")
+
+	formattedOutput := []byte(fmt.Sprintf("%s\n", outputCircle))
+	pipelineStarted := repoNameTestChecker(formattedOutput)
+
+	if pipelineStarted != true {
+		t.Fatal("Not the correct output")
+	}
+
+}
+
+func repoNameTestChecker(output []byte) bool {
+	var f interface{}
+
+	json.Unmarshal(output, &f)
+	m := f.(map[string]interface{})
+
+	x := false
+
+	for key, value := range m {
+
+		if key == "reponame" {
+
+			if value.(string) == "pex-e2e-testing" {
```

Comment:
This test does not belong into this repo.
If you like you can create a test, which checks that the triggerRequest function is called when a postDeployment step is defined and the cluster name matches. And that it is not called when no postDeploymentStep is defined or the cluster name does not match.